tutorials/introduction/06-kolmogorov_equations.jmd

```diff
@@ -0,0 +1,128 @@
+---
+title:  Kolmogorov Backward Equations
+author: Ashutosh Bharambe
+---
+
+```julia
+using Flux, StochasticDiffEq
+using  NeuralNetDiffEq
+using Plots
+using Distributions
+```
+## Introduction on Backward Kolmogorov Equations
+
+The one dimensional backward kolmogorov equation that we are going to deal with is of the form :
+$$
+The backward Kolmogorov Equation deals with a terminal condtion.
+\begin{equation}
+  \frac{\partial p}{\partial t} = \mu(x)\frac{\partial p}{\partial x} + \frac{1}{2}{\sigma^2}(x)\frac{\partial^2 p}{\partial x^2}
+\end{equation}
+$$
+Consider a Black Scholes Partial Differential Equation of the form :
+The Black-Scholes Model governs the price evolution of the European put or call option. In the below equation V is the price of some derivative , S is the Stock Price , r is the risk free interest
+rate and σ the volatility of the stock returns. The payoff at a time T is known to us. And this makes it a terminal PDE.
+$$
+\begin{equation}
+  \frac{\partial V}{\partial t} + rS\frac{\partial V}{\partial S} + \frac{1}{2}{\sigma^2}{S^2}\frac{\partial^2 V}{\partial S^2} -rV = 0
+\end{equation}
+$$
```

Comment:
where... you're not giving the full definition of the PDE. You need to define that it has a valuation at the end.

Reply:
Do I need to explain what is a PDE? or the Black Scholes PDE in detail?

Reply:
No, not "a PDE", "the PDE". You haven't defined the well-defined PDE, in the mathematical sense of "well-defined". There's no domain. If you don't define the terminal condition, the PDE isn't solvable.